src/UXClient/Components/HierarchyNavigation/HierarchyNavigation.ts

```diff
@@ -1201,6 +1198,22 @@ class HierarchyNavigation extends Component{
         return `instance-${Utils.getInstanceKey(instance)}`;
     }
 
+    private instanceNodeStringToDisplay = (instance) => {
+        let str;
+        if (instance.highlights) {
+            if (instance.highlights.name) {
+                str = instance.highlights.name;
```

Comment:
optional chaining somehow?

Reply:
This was my thought here as well, but because there are multiple levels of if / else with different conditions.  It might make it more confusing here.